xarray/tests/test_dataset.py

```diff
@@ -982,9 +983,8 @@ def test_isel_fancy(self):
         self.assertArrayEqual(actual['var2'], expected_var2)
         self.assertArrayEqual(actual['var3'], expected_var3)
 
-    def test_isel_dataarray(self):
+    def array(self):
```

Comment:
Revert? Renaming this disables the unit test.

Reply:
It's my careless.
Fixed.